### PR TITLE
Alex

### PR DESCRIPTION
**Pull request by Alexander-Frost**
Update Documentation.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # Network Handler
 
-I wrote this because I found myself repeating a lot of boilerplate code when sending and receiving data with APIs. Essentially, every time I'd want to make a network call, I'd have to check for errors, check the response code, check if there's data, check if the data is valid, and finally use the data. Every. Time. (Sure, you could skip a couple of those steps occasionally, but then you might run into an unforeseen error).
+NetworkHandler was written to save you time by cutting out the needlessly messy boilerplate code from `UrlSession`. Typically, everytime you make a network call, you have to check for errors, response codes, data existence and data validity. Every. Single.. Time... Skipping those steps while using `UrlSession` might result in unforseen consequences. Luckily, we built `NetworkHandler` as a solution.
```
Comment (mredig) [inline Readme.md]: URLSession

File: Readme.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
-NetworkHandler primarily consists of three main functions:
+NetworkHandler consists of 3 core functions:
 
 * `transferMahOptionalDatas`:
-	* Sometimes you don't care if the server provides data or there may be a situation where the data MIGHT get returned or it MIGHT not, just as long as the transaction was successful otherwise. In these situations, you can use `transferMahOptionalDatas` and it'll provide you with `Data?` when successful, and a `NetworkError` when failure occurs.
+	* Occasionally, you want to make network requests without needing to check for response codes. You use  `transferMahOptionalDatas`  in these situations to provide you strictly with `Data?` when successful and a `NetworkError` when unsuccessful.
```
Comment (mredig) [inline Readme.md]: Double space before "in"

Comment (mredig) [reply Readme.md]: Also, response codes are different. This is when you don't necessarily care if data is returned from the server. Response codes are still checked.

File: Readme.md [hunk 3]
```diff
@@ -110,28 +110,35 @@ There's also built in mocking! Just toggle it on, tell it if you want to get an
...
+    
+* `NetworkHandler`:
+	* The bread to the above butter, this class helps with `URLSession.dataTasks`
 
 ### Installation:
 
 1. Download and install
 	* Carthage
-		* I recommend using Carthage. Simply add the following line to your cartfile. (and of course, follow the remaining carthage instructions as usual)
+		* We recommend using Carthage. Simply add the following line to your cartfile. (and of course, follow the remaining carthage instructions as usual)
```
Comment (mredig) [inline Readme.md]: Carthage is recommended?

Comment (Alexander-Frost) [reply Readme.md]: Carthage is always recommended. Who even uses cocoapods?

Comment (mredig) [reply Readme.md]: I mean that as suggested wording. :)

Comment (Alexander-Frost) [reply Readme.md]: Changed

**Review by mredig — APPROVED** [review]
Great changes! Thanks!

**Pull request merged**